client/src/Component/views/Persona/index.style.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
```

Comment:
somewhere in the persona style, the card height should be set to auto so that the image is not squished.
![Screen Shot 2019-04-28 at 15 23 39](https://user-images.githubusercontent.com/37771591/56865868-b4d9e580-69ca-11e9-96fd-ce8342a4e328.png)
![Screen Shot 2019-04-28 at 15 26 22](https://user-images.githubusercontent.com/37771591/56865871-b73c3f80-69ca-11e9-83dc-acb92b50d882.png)